Put CSS in exports ?

I am trying to make jupyterlab_h5web into a JupyterLab pre-built extension: https://github.com/silx-kit/jupyterlab-h5web/pull/50

Unfortunately, the building of the extension fails because it cannot resolve the `@h5web/app` CSS I need to import:

```
ModuleNotFoundError: Module not found: Error: Package path ./dist/style-lib.css is not exported from package .../jupyterlab-h5web/node_modules/@h5web/app (see exports field in .../jupyterlab-h5web/node_modules/@h5web/app/package.json)
```

<details>
<summary>Error callstack</summary>
<pre>
    at .../jupyterlab-h5web/node_modules/webpack/lib/Compilation.js:2004:28
    at.../jupyterlab-h5web/node_modules/webpack/lib/NormalModuleFactory.js:795:13
    at eval (eval at create (.../jupyterlab-h5web/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:8:1)
    at .../jupyterlab-h5web/node_modules/webpack/lib/NormalModuleFactory.js:275:22
    at eval (eval at create (.../jupyterlab-h5web/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:7:1)
    at .../jupyterlab-h5web/node_modules/webpack/lib/NormalModuleFactory.js:431:22
    at .../jupyterlab-h5web/node_modules/webpack/lib/NormalModuleFactory.js:124:11
    at .../jupyterlab-h5web/node_modules/webpack/lib/NormalModuleFactory.js:667:25
    at .../jupyterlab-h5web/node_modules/webpack/lib/NormalModuleFactory.js:852:8
    at .../jupyterlab-h5web/node_modules/webpack/lib/NormalModuleFactory.js:972:5
.../jupyterlab-h5web/node_modules/@jupyterlab/builder/lib/build-labextension.js:100
</pre>
</details>

Should we include the CSS in the `exports` field of the package ?

Some issues for reference:
- https://github.com/jupyterlab/jupyterlab/issues/9948
- Another package that had the same issue: https://github.com/brianvoe/slim-select/issues/269

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -8,7 +8,6 @@
   - [Icon set](#icon-set)
 - [Build](#build)
   - [Use built packages in demos](#use-built-packages-in-demos)
-  - [Use built packages in other local apps](#use-built-packages-in-other-local-apps)
 - [Code quality](#code-quality)
   - [Fixing and formatting](#fixing-and-formatting)
   - [Editor integration](#editor-integration)

--- a/packages/app/README.md
+++ b/packages/app/README.md
@@ -40,3 +40,11 @@ function MyApp() {
 
 export default MyApp;
 ```
+
+> If your bundler supports it (e.g. webpack 5), you may be able to shorten the
+> stylesheet import paths as follows:
+>
+> ```ts
+> import '@h5web/app/style-lib.css';
+> import '@h5web/app/style.css';
+> ```

--- a/packages/app/package.json
+++ b/packages/app/package.json
@@ -19,6 +19,10 @@
     "module": "dist/index.js",
     "types": "dist/index.d.ts",
     "exports": {
+      "./dist/style-lib.css": "./dist/style-lib.css",
+      "./style-lib.css": "./dist/style-lib.css",
+      "./dist/style.css": "./dist/style.css",
+      "./style.css": "./dist/style.css",
       ".": {
         "require": "./dist/index.cjs",
         "import": "./dist/index.js"

